lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/persistence/DBOMembershipRequestTest.java

```diff
@@ -109,39 +109,6 @@ public void testRoundTrip() throws DatastoreException, NotFoundException, Unsupp
 		assertEquals(clone, clone2);
 	}
 
-	@Test
-	public void testTranslatorRefactor() throws IOException {
-		DBOMembershipRequest backup = new DBOMembershipRequest();
-		String backupProperties =
-				"<MembershipRqstSubmission>\n" +
-				"  <createdOn>2014-02-05 22:21:37.765 UTC</createdOn>\n" +
-				"  <createdBy>1976831</createdBy>\n" +
-				"  <userId>2223382</userId>\n" +
-				"  <teamId>2223746</teamId>\n" +
-				"</MembershipRqstSubmission>\n";
-		backup.setProperties(zip(backupProperties.getBytes()));
-
-		// Method under test
-		DBOMembershipRequest translated = backup.getTranslator().createDatabaseObjectFromBackup(backup);
-
-		String translatedProperties = new String(unzip(translated.getProperties())).trim();
-		// Assert that the top level xml tags are updated
-		assertTrue(translatedProperties.startsWith("<MembershipRequest>"));
-		assertTrue(translatedProperties.endsWith("</MembershipRequest>"));
-
-		String expectedProperties =
-				"<MembershipRequest>\n" +
-				"  <createdOn>2014-02-05 22:21:37.765 UTC</createdOn>\n" +
-				"  <createdBy>1976831</createdBy>\n" +
-				"  <userId>2223382</userId>\n" +
-				"  <teamId>2223746</teamId>\n" +
-				"</MembershipRequest>\n";
-		MembershipRequest expectedDTO = deserialize(zip(expectedProperties.getBytes()));
-		MembershipRequest translatedDTO = deserialize(translated.getProperties());
-		// Assert that the translated DTO's contents are identical to the backup DTO's
-		assertEquals(expectedDTO, translatedDTO);
-	}
-
 	@Test
 	public void testTranslatorRefactorAlreadyTranslated() throws IOException {
```

Comment:
I believe you can now remove ALL tests of the translator, since the translator is now trivial.